creator-node/src/services/stateMachineManager/stateReconciliation/stateReconciliationUtils.js

```diff
@@ -101,6 +101,14 @@ const getNewOrExistingSyncReq = async ({
   return { syncReqToEnqueue }
 }
 
+const getSyncStatusByUuid = async (targetUrl, syncUuid) => {
```

Comment:
nit why put this here vs colocate in syncUtil - no strong opinion just a Q

Reply:
no strong opinion either. my reasoning was to split between issuing and processing.
issuing (all done in `src/services/stateMachineManager`):
* `getSyncStatusByUuid` (this function): fetches the status for the issuing node to read

processing (all done in `src/services/sync`):
* `syncUtil.js` functions: sets redis status on the node that's processing the sync. when it gets the status from redis it's only for the route controller to expose it

Reply:
cool